.github/workflows/update_api.yml

```diff
@@ -0,0 +1,76 @@
+name: Update API 
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "30 3 * * *"
+jobs:
+  update-api:
+    if: ${{ github.repository == 'opensearch-project/opensearch-py' }}
+    runs-on: ubuntu-latest
+    permissions:
```

Comment:
You would have to use `opensearch-trigger-bot` to use these permissions. See https://github.com/opensearch-project/opensearch-py/blob/main/.github/workflows/backport.yml#L16-L22

Reply:
Ah I see you added this on L32, so this permissions block is not needed.